### PR TITLE
Welcome Message for Users

### PR DESCRIPTION
**Pull request by LorenzoFerreyra**
Added a welcome message for users including cripto price and asset symbols

### CODE REVIEW

**Comment by LorenzoFerreyra** [comment]
For next step, we're missing:
*Another message telling users their subscribed assets performance, which could be based on an HTML template with some design.
* This will be sent based on certain fixed period (a number of days chosen by the users).
* We still need a feature that lets users choose this period

File: moneytracker/views.py [hunk 1]
```diff
@@ -36,15 +39,32 @@ def post(self, request, *args, **kwargs):
...
                 PortfolioAsset.objects.create(portfolio=portfolio, asset=asset, quantity=1)
+
+                # we fetch Coinbase API to get asset info and append into the list
+                try:
+                    crypto_data = LiveCryptoData(f'{asset.symbol}').return_data()
+                # Usamos .iloc[-1] para seleccionar el último elemento (la última fila) de un DataFrame de Pandas,
+                # En este caso, nos da acceso al precio más reciente del ticker correspondiente
+                    current_price = crypto_data['price'].iloc[-1]
+                    asset_info_list.append(f'{asset.symbol}: {round(float(current_price), 2)}$')
+                except RequestException as e:
```
Comment (esgaelramos) [inline moneytracker/views.py:54]: Check if we don't use a except for ValueError?

**Review by esgaelramos — CHANGES_REQUESTED** [review]
Great change, small and directly...
My recommendation:
+ Refac for integrate new changes
+ Encapsular this logic in a function???
+ For the tests we need Mock the LiveCrypto and his return (get_value?) and the pd method iloc, haha, it's alot, for that it's better create a new function

**Comment by LorenzoFerreyra** [comment]
Cami also thinks it's a good idea to make all this a unique function and call it appropriately 👍🏼 
We can discuss it now when we meet

**Comment by esgaelramos** [comment]
I need to close this PR:)
The # of conflicts is too long, haha, soo, please develop if is necessaty this feature in a new branch (dont forget update main) thankss:)

**Pull request closed without merging**